test/expect/TestScript.test_lstm_fusion_cuda-forward.expect

```diff
@@ -19,59 +19,68 @@ with prim::DifferentiableGraph_0 = graph(%0 : Float(*, *)
   %8 : Float(*, *) = aten::mm(%5, %7)
   %9 : Float(*, *) = aten::t(%4)
   %10 : Float(*, *) = aten::mm(%3, %9)
-  %11 : int = prim::Constant[value=1]()
-  %12 : int[] = aten::size(%8)
-  %13 : int[] = aten::size(%10)
-  %14 : int[] = aten::size(%2)
-  %15 : Float(*, *), %16 : Float(*, *) = prim::FusionGroup_0(%2, %8, %10)
```

Comment:
Why didn't the two fusion groups get fused together before? I'm trying to think about that but I'm not sure what prevents the fusion

Reply:
Because `aten::size` on chunk outputs prevented us from doing the exchanges that expose larger fusions (we used to check that all users of chunk outputs are in the fusion group).

Reply:
Kind of? We could keep the original chunk and delete it conditionally on the fusion group being the only user, it's true that it would be simpler, but I think it's better to do the `prim::ChunkSizes` trick, as we can avoid allocating all those intermediate chunked tensors, just to check their sizes...

Reply:
Did you mean this check? https://github.com/pytorch/pytorch/blob/33b1f9f71a3d5eec2b2bde694aa77526175fafd0/torch/csrc/jit/passes/graph_fuser.cpp#L604-L612

Reply:
Yup, that one.